app-spec.yml

```diff
@@ -10,6 +10,10 @@ envs:
     scope: RUN_AND_BUILD_TIME
     type: SECRET
     value: ${WEB3_INFURA_PROJECT_ID}
+  - key: AIRTABLE_API_KEY
```

Comment:
I would rather only include this on the `envs` section for the bot that needs the API key - right now only the discount bot uses Airtable